announcing-community-cdn.rst

```diff
@@ -0,0 +1,90 @@
+.. post:: May 12, 2020
+   :tags: community, cdn, hosting
+   :author: Eric
+
+.. meta::
+   :description lang=en:
+
+      Read the Docs Community now has a CDN enabled for all projects.
```

Comment:
This meta description is probably a little too short and probably just won't get used by search engines. It's a pretty [inexact science](https://moz.com/learn/seo/meta-description) whether one gets used but I'd suggest something like this:

```suggestion
      Read the Docs Community now has Cloudflare CDN enabled for all projects resulting in faster and more secure documentation worldwide.
```